plugins/modules/proxmox_sendkey.py

```diff
@@ -0,0 +1,486 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2025, (@miyuk) <enough7531@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: proxmox_sendkey
+short_description: Send key presses to a Proxmox VM console
+version_added: 1.5.0
+description:
+  - Uses the Proxmox API to send a sequence of key presses to the console of a VM.
+  - Keys can be specified explicitly or derived from a plain text string.
+author:
+  - "miyuk (@miyuk172) <enough7531@gmail.com>"
+attributes:
+  check_mode:
+    support: none
+  diff_mode:
+    support: none
+notes:
+  - Keys must match the qemu key names listed in
+    L(QEMU ui.QKeyCode,https://www.qemu.org/docs/master/interop/qemu-qmp-ref.html#enum-QMP-ui.QKeyCode).
+options:
+  name:
+    description:
+      - The unique name of the VM.
+    type: str
+  vmid:
+    description:
+      - The unique ID of the VM.
+      - Takes precedence over O(name).
+    type: int
+  keys_send:
+    description:
+      - List of keys or key sequence to send in order.
+      - Each item must follow the qemu key naming format such as
+        C(ctrl-alt-delete) or C(ret).
+      - You can specify either O(keys_send) or O(string_send) of them.
+    type: list
+    elements: str
+  string_send:
+    description:
+      - Raw string that will be transformed to the corresponding key presses
```

Comment:
```suggestion
      - Raw string that will be transformed to the corresponding key presses
      - Only ASCII-characters are supported.
```
Or some such, looked similar to the ascii keyspace to me. **But I did not take a deeper look into it**